source/BatchDrawList.cpp

```diff
@@ -122,8 +122,9 @@ bool BatchDrawList::Add(const Body &body, Point position, float clip)
 	if(Cull(body, position))
 		return false;
 
+	data[body.GetSprite()].alpha = body.Alpha();
 	// Get the data vector for this particular sprite.
-	vector<float> &v = data[body.GetSprite()];
+	vector<float> &v = data[body.GetSprite()].vertices;
```

Comment:
You're using `data[body.GetSprite()]` twice, let's add an intermediate variable to avoid two lookups.

Reply:
hmmm, isnt it more ram intensive to have another variable?
Is that really an improvement?

Reply:
I mean not really, and in any case rather more RAM than two lookups for *speeeed*

Reply:
memory doesnt matter generally its more about speed

Reply:
doneny, done